py-polars/tests/unit/operations/arithmetic/test_arithmetic.py

```diff
@@ -611,110 +610,6 @@ def test_array_arithmetic_same_size(
     )
 
 
-@pytest.mark.parametrize(
-    ("expected", "expr", "column_names"),
-    [
-        ([[2, 4], [6]], lambda a, b: a + b, ("a", "a")),
-        ([[0, 0], [0]], lambda a, b: a - b, ("a", "a")),
-        ([[1, 4], [9]], lambda a, b: a * b, ("a", "a")),
-        ([[1.0, 1.0], [1.0]], lambda a, b: a / b, ("a", "a")),
-        ([[0, 0], [0]], lambda a, b: a % b, ("a", "a")),
-        (
-            [[3, 4], [7]],
-            lambda a, b: a + b,
-            ("a", "uint8"),
-        ),
-        (
-            [[[2, 4]], [[6]]],
-            lambda a, b: a + b,
-            ("nested", "nested"),
-        ),
-        (
-            [[[2, 4]], [[6]]],
-            lambda a, b: a + b,
-            ("nested", "nested_uint8"),
-        ),
-    ],
-)
-def test_list_arithmetic_same_size(
```

Comment:
I will check these tests back in later

Reply:
Maybe just add comment saying "no need to review these functions, they're cut/pasted from test_arithmetic.py"?

Reply:
I realized I can actually leave those tests here for now instead of removing them